### PR TITLE
Typo fixes - Season 1 Finale

### PR DESCRIPTION
**Pull request by CoffeeBrewer64**
- Removes 2 unused images which were meant to be in the README.md gallery
- Updated the README.md PROPERLY (I know. Shocker.)
- Fixed some logging dialog in some files
- Other typo stuff

### CODE REVIEW

File: lang/en-gb.xml [hunk 1]
```diff
@@ -553,7 +553,7 @@ You can earn trophies by using an application that supports the trophy feature.<
 		<delete_user>Delete User</delete_user>
 		<user_delete>Select the user you want to delete.</user_delete>
 		<user_delete_msg>The following user will be deleted.</user_delete_msg>
-		<user_delete_message>If you delete the user, that user's saved data, trophies will be deleted.</user_delete_message>
+		<user_delete_message>If you delete the user, that user's saved data and trophies will be deleted.</user_delete_message>
```
Comment (Zangetsu38) [inline lang/en-gb.xml:556]: Revert

File: lang/en-gb.xml [hunk 2]
```diff
@@ -580,7 +580,7 @@ After the download is complete, Vita3K will restart automatically and then insta
 		<later_version_already_installed>The later version of Vita3K is already installed.</later_version_already_installed>
 		<latest_version_already_installed>The latest version of Vita3K is already installed.</latest_version_already_installed>
 		<new_features>New Features in Version {}</new_features>
-		<authors>Authors</authors>
+		<authors>Author</authors>
```
Comment (Zangetsu38) [inline lang/en-gb.xml:583]: Revert

Comment (CoffeeBrewer64) [reply lang/en-gb.xml:583]: The actual GUI for updating only shows one author, and "authors" is multiple, so it is correct for there to be no "s".

File: lang/en-gb.xml (hunk 1 above)
Comment (CoffeeBrewer64) [reply lang/en-gb.xml:556]: This is what it looks like without. 
image
it doesn't make sense.

File: lang/en-gb.xml (hunk 2 above)
Comment (Zangetsu38) [reply lang/en-gb.xml:583]: Code show author for all commit, so if have few commit, have few authors

File: lang/en-gb.xml (hunk 1 above)
Comment (Zangetsu38) [reply lang/en-gb.xml:556]: Is ps4 screen, is good

Comment (CoffeeBrewer64) [reply lang/en-gb.xml:556]: I don't understand...

File: lang/en-gb.xml (hunk 2 above)
Comment (CoffeeBrewer64) [reply lang/en-gb.xml:583]: oh right, didnt see that.

File: lang/en-gb.xml (hunk 1 above)
Comment (Zangetsu38) [reply lang/en-gb.xml:556]: This string is from ps4 interface for create user

Comment (CoffeeBrewer64) [reply lang/en-gb.xml:556]: But it doesn't make sense.

Comment (CoffeeBrewer64) [reply lang/en-gb.xml]: fixed

File: lang/en-gb.xml (hunk 2 above)
Comment (CoffeeBrewer64) [reply lang/en-gb.xml]: fixed

**Review by Zangetsu38 — APPROVED** [review]

**Pull request merged**